curriculum/challenges/english/10-coding-interview-prep/take-home-projects/build-a-pinterest-clone.english.md

```diff
@@ -17,8 +17,8 @@ Fulfill the below <a href='https://en.wikipedia.org/wiki/User_story' target='_bl
 <strong>User Story:</strong> As an unauthenticated user, I can browse other users' walls of images.
 <strong>User Story:</strong> As an authenticated user, if I upload an image that is broken, it will be replaced by a placeholder image. (can use jQuery broken image detection)
 <strong>Hint:</strong> <a href='http://masonry.desandro.com/' target='_blank'>Masonry.js</a> is a library that allows for Pinterest-style image grids.
```

Comment:
This `http` link does redirect to the `https` version, but fixing it will make the linked page load faster;
```suggestion
<strong>Hint:</strong> <a href='https://masonry.desandro.com/' target='_blank'>Masonry.js</a> is a library that allows for Pinterest-style image grids.
```